mesonbuild/programs.py

```diff
@@ -340,6 +340,12 @@ class OverrideProgram(ExternalProgram):
 
     """A script overriding a program."""
 
+    def __init__(self, *args: T.Any, version: str, **kwargs: T.Any) -> None:
+        self.version = version
```

Comment:
The parent class already defines self.cached_version so perhaps we should reuse that here. :)

Reply:
Thanks!